from django.apps import AppConfig


class CalloverexceptionreviewConfig(AppConfig):
    name = 'calloverexceptionreview'
